packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -360,9 +360,23 @@ class FlutterInspectorSettingsDialog extends StatelessWidget {
               'Widgets in these directories will show up in your summary tree.',
               style: theme.subtleTextStyle,
             ),
-            Text(
-              '(e.g. /absolute/path/to/myPackage)',
-              style: theme.subtleTextStyle,
+            RichText(
+              text: TextSpan(
+                text: '(e.g. /absolute/path/to/myPackage) ',
+                style: theme.subtleTextStyle,
+                children: [
+                  LinkTextSpan(
+                    context: context,
+                    link: Link(
+                      display: 'Learn more',
+                      url:
+                          'https://docs.flutter.dev/tools/devtools/inspector#package-directories',
+                      gaScreenName: InspectorScreen.id,
+                      gaSelectedItemDescription: gac.documentationLink,
+                    ),
+                  ),
+                ],
+              ),
```

Comment:
you can leave the Text widget as it was before, and then add the docs using the MoreInfoLink widget that we use elsewhere in devtools